src/lib/clients/IOClients/apps/Builder.ts

```diff
@@ -14,13 +14,26 @@ export interface RequestParams {
   skipSemVerEnsure?: boolean
 }
 
+export interface BuilderHubRequestOptions {
+  params?: RequestParams
+  headers?: Record<string, any>
+}
+
 export interface BuildResult {
   availability?: AvailabilityResponse
   code?: string
   message?: any
   timeNano?: number
 }
 
+export interface PrepareRequest {
```

Comment:
This is used only on projectUploader, right? You can move it there, don't need to export it

Reply:
Sure!